.github/workflows/build.yml

```diff
@@ -102,6 +102,8 @@ jobs:
             uarch-ram.bin
             uarch-pristine-ram.c
             uarch-pristine-hash.c
+            machine-c-version.h
+            add-generated-files.diff
```

Comment:
Why not `.patch` instead of `.diff` ? Usually the name extensions is `.patch` when packaging in most Linux distros.